src/mozanalysis/stats/bayesian_binary.py

```diff
@@ -0,0 +1,213 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import pandas as pd
+import numpy as np
+import scipy.stats as st
+
+import mozanalysis.stats.summarize_samples as masss
+
+
+def compare_branches(
+    df, col_label, ref_branch_label='control', num_samples=10000,
+    individual_summary_quantiles=masss.default_quantiles,
+    comparative_summary_quantiles=masss.default_quantiles
+):
+    """Jointly sample conversion rates for branches then compare them.
+
+    See `compare_branches_from_agg` for more details.
+
+    Args:
+        df: a pandas DataFrame of queried experiment data in the
+            standard format.
+        col_label (str): Label for the df column contaning the metric
+            to be analyzed.
+        ref_branch_label (str, optional): String in `df['branch']` that
+            identifies the the branch with respect to which we want to
+            calculate uplifts - usually the control branch.
+        num_samples (int, optional): The number of samples to compute.
```

Comment:
Is this the number of bootstrap replicates? 10,000 seems like a very large default.

Reply:
1) Yes.

2) I guess it's worth running some more examples and having discussions re: how much precision we want.